astropy/cosmology/flrw/base.py

```diff
@@ -1096,7 +1146,9 @@ def _integral_comoving_distance_z1z2(self, z1, z2):
         d : `~astropy.units.Quantity` ['length']
             Comoving distance in Mpc between each input redshift.
         """
-        return self._hubble_distance * self._integral_comoving_distance_z1z2_scalar(z1, z2)
+        return self._hubble_distance * self._integral_comoving_distance_z1z2_scalar(
+            z1, z2
+        )
```

Comment:
Don't love this.

Reply:
I could add a `# fmt: skip` and make it one line?

Reply:
I think so. At least for now.